tailwind.config.js

```diff
@@ -29,6 +29,9 @@ module.exports = {
       },
     },
   },
-  variants: {},
+  variants: {
```

Comment:
I didn't figure out what happens in storybook with the `group-hover`

Reply:
I don't understand your comment ? Is the group-hover functionality not working in your storybook ? Or is there something else ?

Reply:
Yes what should happen on group hover ?

Reply:
In facet item, both label and count should darken independently of which one is hovered, and the minus or check at the end of the line should appear.
In facet block, both title and chevron should darken.

Reply:
Ok but why do you need to add TW variants for that ? It's just a question I didn't work on TW configuration.